pkg/services/folder/folderimpl/folder.go

```diff
@@ -185,6 +189,10 @@ func (s *Service) GetChildren(ctx context.Context, cmd *folder.GetChildrenQuery)
 		return nil, folder.ErrBadRequest.Errorf("missing signed in user")
 	}
 
+	if cmd.UID == folder.SharedFolderUID {
```

Comment:
same as above, we could also hide it behind the `nestedFolders` feature flag (no point to return this if it's not enabled)